RacingCar/Calculator.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Calculator.swift
+//  RacingCar
+//
+//  Created by nylah.j on 2022/04/29.
+//
+
+import Foundation
+
+struct Calculator {
```

Comment:
```suggestion
struct StringCalculator {
```
문자열 계산기니 `StringCalculator`는 어떨까라는 의견드려봅니다.